ccpp/data/GFS_typedefs.meta

```diff
@@ -6938,6 +6938,20 @@
   dimensions = (horizontal_loop_extent,vertical_dimension)
   type = real
   kind = kind_phys
+[dkt]
+  standard_name = atmosphere_heat_diffusivity
+  long_name = diffusivity for heat
```

Comment:
Maybe it is good to have the long_name be consistent with ExtDiag(idx)%desc, but we can do this later.